recipes/kadi-apy/meta.yaml

```
@@ -0,0 +1,54 @@
+{% set name = "kadi-apy" %}
+{% set version = "0.46.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://gitlab.com/iam-cms/kadi-apy/-/archive/v{{ version }}/kadi-apy-v{{ version }}.tar.gz
+  sha256: 2a5110f5230a7fe579b79262f3453cb66e780c9bc812cdac2b8b782cb0534ec7
+
+build:
```

Looks like this is missing:

```suggestion
build:
  entry_points:
   - kadi-apy = "adi_apy.cli.main:kadi_apy
```

Also, please add a CLI test for it.